packages/patterns/src/patterns/patternMatchers.js

```diff
@@ -958,7 +966,8 @@ const makePatternKit = () => {
       const { stringLengthLimit } = limit(limits);
       return (
         checkKind(specimen, 'string', check) &&
-        (specimen.length <= stringLengthLimit ||
+        // eslint-disable-next-line prettier/prettier -- JSDoc requires these parentheses!
```

Comment:
bumping Prettier 2.8.0 to 2.8.4 might fix this. I don't see that in the [changelog](https://github.com/prettier/prettier/blob/main/CHANGELOG.md) but I feel like I've stopped running into this problem in agoric-sdk since a recent bump